docs/src/userguide/extension_types.rst

```diff
@@ -453,26 +664,29 @@ definition, for example,::
     cdef class A(B):
         # attributes and methods
 
+.. note::
+
+    Cython currently does not support Forward-declaring extension types
+    in Pure Python mode.
+
```

Comment:
This note should either be left out, or explain that this is not a draw-back because forward-declaring types is not necessary at all. As it stands, it reads like a missing feature.

Reply:
I have removed the note.